concordia/admin/views.py

```diff
@@ -192,27 +208,30 @@ def admin_bulk_import_review(request):
                             continue
 
                         try:
+                            urls.append(url)
+                            url_counter = url_counter + 1
 
-                            count = event_loop.run_until_complete(
-                                import_item_count_from_url(url)
-                            )
-                            total_count = total_count + count
+                            if url_counter == 30:
+                                all_urls.append(urls)
+                                url_counter = 0
+                                urls = []
 
-                            messages.info(
-                                request,
-                                f"{project_title} - [{url}] : {count}",
-                            )
                         except Exception as exc:
                             messages.error(
                                 request,
                                 f"Unhandled error attempting to count {url}: {exc}",
                             )
+
+                all_urls.append(urls)
+                for i, val in enumerate(all_urls):
+                    return_result = fetch_all_urls(val)
```

Comment:
Since the return type is not obvious I would unpack this into logical variables:

```python
message, count = fetch_all_urls(val)
```